test/Microsoft.NET.Build.Tests/RoslynBuildTaskTests.cs

```diff
@@ -51,6 +53,14 @@ public void FullMSBuild_NonSdkStyle(bool useSharedCompilation, Language language
         VerifyCompiler(buildCommand, FxCompilerFileName(language), useSharedCompilation);
     }
 
+    [FullMSBuildOnlyTheory, CombinatorialData]
+    public void FullMSBuild_SdkStyle_ToolsetPackage(bool useSharedCompilation, Language language)
```

Comment:
This is currently failing with

```
D:\sdk-7\artifacts\.nuget\packages\microsoft.net.compilers.toolset\5.0.0-1.25256.101\tasks\net472\Microsoft.CSharp.Core.targets(84,5): error : The application to execute does not exist: 'D:\sdk-7\artifacts\.nuget\packages\microsoft.net.compilers.toolset\5.0.0-1.25256.101\tasks\net472\../bincore\csc.dll' [D:\sdk-7\artifacts\tmp\Debug\FullMSBuild_S---E359EDB9\App1\App1.csproj]
```

until https://github.com/dotnet/roslyn/pull/78402 flows here (that should hopefully fix this)